Hestia/app/src/androidTest/java/com/rugged/application/hestia/UI/dialogs/ChangeCredentialsDialogTest.java

```diff
@@ -0,0 +1,106 @@
+package com.rugged.application.hestia.UI.dialogs;
+
+import android.support.test.espresso.intent.rule.IntentsTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.rugged.application.hestia.R;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import hestia.UI.activities.home.HomeActivity;
+import hestia.UI.activities.login.LoginActivity;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.clearText;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.intent.Intents.intending;
+import static android.support.test.espresso.intent.matcher.ComponentNameMatchers.hasShortClassName;
+import static android.support.test.espresso.intent.matcher.IntentMatchers.hasComponent;
+import static android.support.test.espresso.intent.matcher.IntentMatchers.toPackage;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static com.rugged.application.hestia.R.id.context_menu;
+import static org.hamcrest.Matchers.allOf;
+
+@RunWith(AndroidJUnit4.class)
+public class ChangeCredentialsDialogTest {
+    private final String OLD_USER = "admin";
+    private final String OLD_PASS = "password";
+    private final String NEW_USER = "testuser";
+    private final String NEW_PASS = "testpass";
+    private final String LOGOUT_TEXT = HomeActivity.logoutText;
+    private final String PACKAGE_NAME = "com.rugged.application.hestia";
+    private final String CHANGE_TEXT = HomeActivity.changeCredentialsText;
+
+    @Rule
+    public IntentsTestRule<HomeActivity> mIntentsRule =
+            new IntentsTestRule<>(HomeActivity.class);
+    @Before
+    public void setUp(){
+        // To be sure that the correct standard credentials are set, logout and login.
+        onView(withId(context_menu)).perform(click());
+
```

Comment:
Why an empty line between every command? Not a proper way of making your code more readable. Edit your IDE settings if you need more space between lines.